apps/backoffice-v2/src/pages/KybAndOwnership/components/KybAndOwnershipAssessmentStatus/hooks/useUpdateKybAndOwnershipAssessmentStatus/useUpdateKybAndOwnershipAssessmentStatus.tsx

```diff
@@ -0,0 +1,49 @@
+import { t } from 'i18next';
+import { toast } from 'sonner';
+import type { UpdateableAssessmentStatus } from '@ballerine/common';
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+
+import { HttpError } from '@/common/errors/http-error';
+import { updateAssessmentStatus } from '../../fetchers';
+
+export const useUpdateKybAndOwnershipAssessmentStatus = ({
```

Comment:
Same as the fetchers, I think having this mutation under the assessments domain makes sense.
```suggestion
export const useUpdateKybAndOwnershipAssessmentStatusMutation = ({
```